corehq/apps/export/esaccessors.py

```diff
@@ -26,6 +27,12 @@ def get_case_export_base_query(domain, case_type):
             .sort("opened_on"))
 
 
+def get_sms_export_base_query(domain):
+    return (SMSES()
+            .domain(domain)
+            .sort("date"))
```

Comment:
hmm, ok. there's not too many of them anyway, maybe it's easier to just exclude them when building the export?